atc/syslog/drainer.go

```diff
@@ -115,3 +102,74 @@ func (d *drainer) drainBuild(logger lager.Logger, build db.Build, syslog *Syslog
 
 	return nil
 }
+
+func (d *drainer) sendEvent(logger lager.Logger, build db.Build, syslog *Syslog, ev event.Envelope) error {
+	var (
+		hostname string = d.hostname
+		ts       time.Time
+		tag      string
+		message  string
+	)
+
+	switch ev.Event {
```

Comment:
I think we may also want to log the following events:

* EventTypeInitializeGet
* EventTypeInitializePut
* EventTypeInitializeTask
* EventTypeInitialize
* EventTypeStart
* EventTypeFinish
* EventTypeError

Reply:
Yeah, that make sense, will add more eventType.

Reply:
@evanchaoli just making sure you're fine with these not getting logged:

- EventTypeWaitingForWorker
- EventTypeFinishTask
- EventTypeStartGet
- EventTypeStartPut
- EventTypeSetPipelineChanged
- EventTypeStart
- EventTypeFinish
- EventTypeImageCheck
- EventTypeImageGet